northstar/src/runtime/state.rs

```diff
@@ -206,6 +208,8 @@ impl State {
             config: config.clone(),
             minijail,
             mount_control: MountControl::new(&config).await.map_err(Error::Mount)?,
+            #[cfg(debug_assertions)]
+            internal_repository,
```

Comment:
why do we need to keep this as a field in the struct?

Reply:
Because the directory is removed when the enclosing `Tempdir` is dropped: https://docs.rs/tempfile/3.2.0/tempfile/struct.TempDir.html